NthKey/Model/SeedManager.swift

```diff
@@ -53,13 +53,9 @@ struct SeedManager {
             precondition(false)
         }
         let seedHex = BIP39Mnemonic(entropy)!.seedHex()
+        let masterKey = HDKey(seedHex, .testnet)! // FIXME: HDKey in this case shouldn't depend on network, because fingerprint is unique for the device
```

Comment:
The HDKey object does depend on the network, but its fingerprint doesn't (please check). For the purpose of this function, using `.mainnet` looks better.
https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#master-key-generation

The differentiation between mainnet and testnet happens at depth 2: e.g. `m/84'/0'` (mainnet) vs `m/84'/1'` (testnet), but this doesn't really matter in practice.

Reply:
@Sjors Yes, the fingerprint is the same, but HDKey is network-relewant, just check it. Okay, we'll use `.mainnet` here.